packages/vats/src/core/startWalletFactory.js

```diff
@@ -238,18 +217,45 @@ export const startWalletFactory = async (
       economicCommitteeCreatorFacet,
     },
   );
+  provisionPoolStartResult.resolve(ppFacets);
   instanceProduce.provisionPool.resolve(ppFacets.instance);
 
-  provisionPoolStartResult.resolve(ppFacets);
+  const terms = await deeplyFulfilled(
+    harden({
+      agoricNames,
+      board,
+      assetPublisher: Far('AssetPublisher', {
+        getAssetSubscription: () => E(poolBank).getAssetSubscription(),
+      }),
+    }),
+  );
+  /** @type {WalletFactoryStartResult} */
+  const wfFacets = await E(zoe).startInstance(
+    walletFactory,
+    { Fee: feeIssuer },
+    terms,
+    {
+      storageNode: walletStorageNode,
+      walletBridgeManager,
+      walletReviver: ppFacets.walletReviver,
```

Comment:
I think `ppFacets` doesn't have `walletReviver` at this point.

The facets returned from `E(zoe).startInstance()` are just `publicFacet` and `creatorFacet`. `startGovernedInstance` adds a few more, but I don't see how it could access exo facets such as `walletReviver`.

I think we need a `E(creatorFacet).getWalletReviver()` to make up the gap.

I wonder whether a bootstrap test for this is cost-effective.

Reply:
> I wonder whether a bootstrap test for this is cost-effective.

Yes, definitely! Is there a good example to base it on?

Reply:
How about refining `test-vaults-integration.js`?

Tweak the start-up to pass in some addresses and add another test.serial to consume them.